### PR TITLE
Allow multiple response types.

### PR DESCRIPTION
**Pull request by tcmitche**
Used the existing `makeExamplesOf` function, seems this was the intent
all along.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -104,11 +104,7 @@ function flattenMethods(methods) {
       obj.method = objForMethod.method;
       obj.headers = objForCode.headers;
       obj.description = objForCode.description;
-      _.forEach(objForCode.body, function(objForRespType, respType) {
-        obj.type = respType;
-        obj.example = objForRespType.example;
-        obj.schema = objForRespType.schema;
-      });
+      obj.examples = makeExamplesOf(objForCode);
```
Comment (rdenmark) [inline src/main.js]: Mock is my co-pilot

File: templates/resource.handlebars [hunk 2]
```diff
@@ -66,14 +66,16 @@
...
+      {{#if examples}}
+        {{#each examples}}
+          <h5>
+            Response Example
+            {{#if type}}
+              (<tt>{{type}}</tt>)
+            {{/if}}
+          </h5>
+          {{showCode example type=type}}
+          {{/each}}
```
Comment (rdenmark) [inline templates/resource.handlebars]: Indentation is off.

**Comment by tcmitche** [comment]
@rdenmark Fixed the indentation.

**Comment by rdenmark** [comment]
LGTM

File: src/main.js (hunk 1 above)
Comment (wavebeem) [inline src/main.js]: I am pleased that the solution was just using my own function more.

**Pull request merged**